R/ingest-scp.R

```diff
@@ -1,7 +1,7 @@
-library(pak)
-pak("RPostgres")
-library("dplyr")
-library("DBI")
+library(RPostgres)
+library(dplyr)
+library(DBI)
+library(purrr)
 
```

Comment:
Make sure to use `pak::pak()` only for installing libraries, otherwise use `library()`. As this file evolves into a package function, we'll remove `library()` altogether. I added purrr as well.